charts/nginx-gateway-fabric/templates/rbac.yaml

```diff
@@ -8,15 +8,15 @@ metadata:
   annotations:
     {{- toYaml .Values.serviceAccount.annotations | nindent 4 }}
   {{- if or .Values.serviceAccount.imagePullSecret .Values.serviceAccount.imagePullSecrets }}
-  imagePullSecrets:
-    {{- if .Values.serviceAccount.imagePullSecret }}
-    - name: {{ .Values.serviceAccount.imagePullSecret }}
-    {{- end }}
-    {{- if .Values.serviceAccount.imagePullSecrets }}
-    {{- range .Values.serviceAccount.imagePullSecrets }}
-    - name: {{ . }}
-    {{- end }}
-    {{- end }}
+imagePullSecrets:
+  {{- if .Values.serviceAccount.imagePullSecret }}
+  - name: {{ .Values.serviceAccount.imagePullSecret }}
+  {{- end }}
+  {{- if .Values.serviceAccount.imagePullSecrets }}
+  {{- range .Values.serviceAccount.imagePullSecrets }}
+  - name: {{ . }}
+  {{- end }}
```

Comment:
Does this indentation with the tripple {{- end }} look right? I'm not familiar with style.

Reply:
Yeah, I'd also de-indent the `if` right before the imagePullSecrets, and the last `end` to be flush with the left side.

Reply:
So that it looks like 

```
imagePullSecrets:
{{- if .Values.serviceAccount.imagePullSecret }}
  - name: {{ .Values.serviceAccount.imagePullSecret }}
  {{- end }}
  {{- if .Values.serviceAccount.imagePullSecrets }}
  {{- range .Values.serviceAccount.imagePullSecrets }}
  - name: {{ . }}
  {{- end }}
  {{- end }}
{{- end }}

```

Or by "before" do you mean above which is in the metadata?

Reply:
The one that says `{{- if or .Values.serviceAccount.imagePullSecret .Values.serviceAccount.imagePullSecrets }}`